.golangci.yaml

```diff
@@ -24,12 +24,18 @@ linters-settings:
   gosec:
     excludes:
       - G404 # Insecure random number source (rand)
+  errcheck:
+    exclude-functions:
+      - io.Copy(os.Stdout)
+      - (*github.com/peterstace/simplefeatures/rtree.RTree).RangeSearch
+      - (*github.com/peterstace/simplefeatures/rtree.RTree).PrioritySearch
```

Comment:
These are used in enough places without needing error checks that I think it's best to just disable for these methods.